Adding worksheet_05

akshatgurbuxani opened the pull request.

gallettilance reviewed (changes requested):
nice work! please move this to the `student_worksheets` folder

akshatgurbuxani commented:
Hi Professor, I've moved the file to "student-worksheets" folder. Moreover, I've replaced my files in "lecture_05" folder with the original files. Thanks!
